nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/ConsumeAzureEventHub.java

```diff
@@ -332,6 +345,26 @@ public void setWriterFactory(RecordSetWriterFactory writerFactory) {
                     .valid(false)
                     .build());
         }
+
+        if (storageAccountKey == null && storageSasToken == null) {
+            results.add(new ValidationResult.Builder()
+                    .subject(String.format("%s or %s",
+                            STORAGE_ACCOUNT_KEY.getDisplayName(), STORAGE_SAS_TOKEN.getDisplayName()))
+                    .explanation(String.format("either %s or %s should be set.",
+                            STORAGE_ACCOUNT_KEY.getDisplayName(), STORAGE_SAS_TOKEN.getDisplayName()))
+                    .valid(false)
+                    .build());
+        }
+
+        if (storageAccountKey != null && storageSasToken != null) {
```

Comment:
Should this use StringUtils.isNotBlank() to confirm that the properties are both not null and not blank?

Reply:
Thanks @exceptionfactory code has been modified accordingly.